source/index.html.md

```diff
@@ -50,6 +50,44 @@ This page will be invoked via redirect from the Authorization server at the conc
 
 The other content you see in the source folder is the site for this tutorial. We used <a href="https://github.com/lord/slate" target="_blank">Slate</a> to create the documentation for this tutorial.
 
```

Comment:
Let's move MPages Integration section after this section "Test your App."  I want to introduce MPages integration after they have the SMART app up and running first.

Reply:
Addressed here: https://github.com/cerner/smart-on-fhir-tutorial/pull/29/commits/abe972f5bd1aac07b3886f4504826beedfdf59eb